proversity_reports_script/report_backend/completion_report.py

```diff
@@ -130,6 +138,7 @@ def upload_file_to_storage(self, course, path_file):
             )
         )
 
+
```

Comment:
remove this empty space

Reply:
Addressed, but i think that two blank lines is better for code reading, although pep8 says the opposite.